api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -618,4 +623,15 @@
      */
     <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 
+
+    /**
+     *
+     * Returns an instance of {@link Event} whose event type is Object and without qualifier.
+     * It allows typesafe synchronous or asynchronous event firing without injection of {@link Event} built-in bean requirement.
+     *
+     * @return a new {@link Event} object whose event type is Object and without qualifier
+     * @since 2.0
+     */
+    Event<Object> getEvent();
```

Comment:
I think we might consider adding something like `selectEvent(Class<T> eventType, Annotation... qualifiers)` instead, because users will mostly call something like `getEvent().select(Foo.class).fire(new Foo())` anyway...